src/app/components/Container.tsx

```diff
@@ -23,7 +24,13 @@ function Container({ children, maxWidth = "lg" }: Props) {
   };
 
   return (
-    <div className={`container ${getMaxWidthClass(maxWidth)} mx-auto px-4`}>
+    <div
+      className={`container mx-auto px-4 ${getMaxWidthClass(maxWidth)} ${
+        isScreenView
+          ? "h-full flex flex-col justify-between overflow-y-auto no-scrollbar"
```

Comment:
no way we can avoid this two cases here? 
how did we do it before, or on the receive page?

Reply:
The problem is, right now, we want to push the ConfirmOrCancel to the bottom, and for that, we're using flex and justify-between vertically apart from that, there's nothing new (except for h-full to make sure the height used for distribution is the entire space available)

Reply:
cc @escapedcat so maybe if that "pushing the ConfirmOrCancel to the bottom" is too complicated, we should rethink/drop that at some point.

Reply:
Maybe we eventually will end up with a sticky bottom-bar for that? I'll mention this in our next ui/ux call